SmartDeviceLink/SDLAudioStreamManager.h

```diff
@@ -66,6 +66,13 @@ typedef NS_ENUM(NSInteger, SDLAudioStreamManagerError) {
  */
 - (void)pushWithFileURL:(NSURL *)fileURL;
 
+/**
+ Push a new audio buffer onto the queue. Call `playNextWhenReady` to start playing the pushed audio buffer.
+
+ @param data The audio buffer to be pushed onto the queue
+ */
+- (void)pushWithData:(NSData *)data;
```

Comment:
The documentation should note the required format:
> Format: PCM
Sample Rate: 16k
Number of Channels: 1
Bits Per Second (BPS): 16 bits per sample / 2 bytes per sample

Reply:
Fixed.

I considered the deprecation, but I didn't want to force developers to use the `push` methodology which waits until the previous one finishes before playing the next one.